site/_includes/layouts/extension-samples.njk

```diff
@@ -0,0 +1,194 @@
+{% extends "layouts/base.njk" %}
+
+{% block css %}
+  {% InlineCss '/css/extension-samples.css' %}
+  {# unique styles, needed only for extension samples hero card. #}
+  <style>
+    @media screen and (min-width: 860px) {
+      .hero-card {
+        padding-top: 1.75rem;
+        padding-bottom: 1.75rem;
+      }
+    }
+
+    .hero-card__inner {
+      display: block;
+    }
+
+    @media (min-width: 1231px) {
+      .hero-card__inner {
+        display: grid;
+        grid-template-columns: 2fr 3fr;
+      }
+    }
+
+    .hero-card h1 {
+      font-size: 2em;
+    }
+  </style>
+{% endblock %}
```

Comment:
Nit, but since this page already as a specific stylesheet in /css/extension-samples.css, just append the rules there. Makes one `<style>` tag less.